translate/tools/pocount.py

```diff
@@ -46,6 +46,38 @@
 default_style = style_full
 
 
+class ConsoleColor:
+    """Class to implement color mode.
+    """
+    # print using color? Default to true
+    color_mode = True
+    COLOR_PURPLE = "\033[95m"
+    COLOR_GREEN = "\033[92m"
+    COLOR_YELLOW = "\033[93m"
+    COLOR_RED = "\033[91m"
+    COLOR_DEFAULT = "\033[0m"
+
+    @staticmethod
+    def HEADER():
+        return COLOR_PURPLE if ConsoleColor.color_mode else ""
```

Comment:
I tried running this PR locally, but it is not able to find `COLOR_PURPLE` (`__main__: ERROR: global name 'COLOR_PURPLE' is not defined`). This can be fixed if this method is changed like:

```python
    @classmethod
    def HEADER(cls):
        return cls.COLOR_PURPLE if ConsoleColor.color_mode else ""
```

Note the usage of `@classmethod` and `cls` to reference the class.

The same happens for all the other methods in this class, so please adjust them the same way.